explorer/src/lib/components/blocks-card/BlocksCard.svelte

```diff
@@ -17,10 +14,35 @@
   /** @type {Boolean} */
   export let loading;
 
+  const ITEMS_TO_DISPLAY = 15;
+
+  let itemsToDisplay = ITEMS_TO_DISPLAY;
+
   /** @type {number} */
   let clientWidth;
 
-  $: classes = makeClassName(["blocks-card", className]);
+  /** @type {Block[]}*/
+  let displayedBlocks;
+
+  /** @type {Boolean}*/
+  let disabledLoading = false;
```

Comment:
I think this variable deserves a better name, to clearly indicate its purpose: something like `isLoadMoreDisabled`?